device/bluetooth_mapper/configuration/config.go

```diff
@@ -182,7 +184,7 @@ func (b *BLEConfig) Load() error {
 			if strings.ToUpper(deviceModel.Name) == strings.ToUpper(b.Device.Name) {
 				for _, property := range deviceModel.Properties {
 					if strings.ToUpper(property.Name) == strings.ToUpper(actionConfig.PropertyName) {
-						if strings.ToUpper(property.AccessMode) == "RW" {
+						if strings.ToUpper(property.AccessMode) == strings.ToUpper(READWRITE){
```

Comment:
can't we just compare without comparing to upper case ? since our schema is defined , users are expected to provide the correct enum values.

Reply:
Removed case insensitive check and replaced it with a case sensitive one. Going forward when we have API validation we can prevent the users from accidentally giving the wrong case for enum values, thus eliminating the need for a case insensitive check. Thanks